.github/workflows/force_release.yml

```diff
@@ -0,0 +1,117 @@
+name: Force Release
```

Comment:
This is unchanged from the manually-trigger-release branch, no need to review closely